projects/core/src/lib/components/toolbar/measure/measure.component.spec.ts

```diff
@@ -57,42 +46,8 @@ describe('MeasureComponent', () => {
     const { mapServiceMock } = await setup();
     expect(screen.getByLabelText('Measure distance'));
     expect(screen.getByLabelText('Measure area'));
-    expect(mapServiceMock.mapService.createTooltip$).toHaveBeenCalled();
     expect(mapServiceMock.mapService.renderFeatures$).toHaveBeenCalled();
     expect(mapServiceMock.mapService.createTool$).toHaveBeenCalled();
   });
 
-  test('enables tool, acts on drawing, disables tool', async () => {
```

Comment:
Removed this test because it was very dependant on the tooltip which we no longer have